src/components/Analysis/Modes/LoadedModesList.jsx

```diff
@@ -43,6 +43,8 @@ const LoadedModesList = ({ timeseries }) => {
     if (mode) {
       return transformTimeseriesToTrips(
         timeseriesSortedByModes[mode].timeseries
+      ).sort((a, b) =>
+        a.properties.start_fmt_time.localeCompare(b.properties.start_fmt_time)
```

Comment:
Personnally, I would move this code into `transformTimeseriesToTrips`and rename the function into `transformTimeseriesToSortedTrips`

Reply:
and that would help to test the sorting feature

Reply:
another reason is: this code is duplicated in loaded purpose list

Reply:
Vu, traité dans la prochaine PR 👍